rest_framework_json_api/utils.py

```diff
@@ -186,6 +186,14 @@ def get_related_resource_type(relation):
         relation_model = relation.model
     elif hasattr(relation, 'get_queryset') and relation.get_queryset() is not None:
         relation_model = relation.get_queryset().model
+    elif (
+            getattr(relation, 'many', False) and
+            hasattr(relation.child, 'Meta') and
```

Comment:
I don't quite understand.  If the serializer may not be a `ModelSerializer`, then shouldn't we *not* assume that `Meta` has a `model`?